indigo/analysis/refs/base.py

```diff
@@ -91,44 +92,45 @@ def setup(self, root):
         super().setup(root)
 
     def setup_subtypes(self):
-        self.subtypes = [s for s in Subtype.objects.all()]
-        self.subtype_names = [s.name for s in self.subtypes]
-        self.subtype_abbreviations = [s.abbreviation for s in self.subtypes]
-
-        self.subtypes_string = '|'.join([re.escape(s) for s in self.subtype_names + self.subtype_abbreviations])
+        self.full_subtypes = Subtype.objects.all()
+        self.subtypes = [s.name for s in self.full_subtypes] + \
+                        [s.abbreviation for s in self.full_subtypes]
```

Comment:
Given what we've learnt from term matching and italics, does it make sense to sort these longest first?